### DISCUSSION TITLE
from crossfilter.js to mosaic

### DISCUSSION THREAD

**Discussion by v4lue4dded** [uwdata/mosaic#505] [Ideas]
So far I've been using the crossfilter (https://github.com/crossfilter/crossfilter) library for my visualizations.
Which is great since it is easy to use with dc:
https://dc-js.github.io/dc.js/
and c3:
https://drarmstr.github.io/chartcollection/examples/#worldbank 

Which is how I am currently working on my global burden of disease dashboard:
https://v4lue4dded.github.io/GBD/modern_gbd_v09.html

And I'm considering switching to mosaic.

It would be really cool if there was some tutorial/blog post/readme page telling users that want to switch from crossfilter to mosaic how to best do that, since crossfilter is still the industry standard as far as I can tell.
Does such a resource to make a transition easier already exist?

**Comment by domoritz** [comment]
Great idea. There is no such tutorial as far as I know so it would be great if you could make one.
